closure/goog/i18n/numberformat.js

```diff
@@ -580,6 +588,140 @@ goog.i18n.NumberFormat.prototype.roundNumber_ = function(number) {
 
 
 /**
+ * This function is called when the length of the repeating digits is at least
```

Comment:
Method descriptions should start with a sentence written in the third person declarative voice. E.g., "Formats blah blah blah...".